teuthology/suite/run.py

```diff
@@ -29,6 +29,7 @@
 class Run(object):
     WAIT_MAX_JOB_TIME = 30 * 60
     WAIT_PAUSE = 5 * 60
+    JOBS_TO_SCHEDULE_THRESHOLD = 500
```

Comment:
JOB_THRESHOLD would be concise I guess :-)

Reply:
I don't see an issue with a more explicit name, which is exactly what you wanted earlier :)

Reply:
I prefer some kind of balance between explicit and cryptic, like concise :-D cheers

Reply:
@neha-ojha anyways, seems like my comments are getting out of constructive way, go ahead and merge if the functionality is fine, I'll raise another follow up PR with improvement :-)

Reply:
@kshtsk The tests are passing https://github.com/ceph/teuthology/pull/1657#issuecomment-878474666 and my local testing looks good as well https://github.com/ceph/teuthology/pull/1657#issue-682088773. I am going to go ahead and merge this version, please feel free to update the flag/variable names as per your liking. Thanks!